lib/mongoid/fields/field_types.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module Mongoid
+  module Fields
+
+    # Singleton module which contains a cache for field type definitions.
```

Comment:
This is a mapping of field type string/symbol aliases to the underlying classes, not a cache.

Reply:
Done.